### PR TITLE
Updated Ovni

### PR DESCRIPTION
**Pull request by lleftTheDragon**
Massive update to Ovni

## About the PR 

**Media**
Screenshot_804
Screenshot_805
Screenshot_814

Screenshot_807
Screenshot_808
Screenshot_809
Screenshot_810
Screenshot_811
Screenshot_812

Screenshot_813

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

Added reverse engineering machine in engineering

Big tree removed due to excessive entrapment, replaced with a stage.

Security outpost removed and replaced with a library with a DnD table.

Second bar moved to maints (added flair to second bar)

Second bar near HOP replaced with EVA room

Two reinforced windows in arrivals replaced with secure windoors accessible by engineering, access finally granted to the APC.

Cargo request computer in bridge replaced with Shipyard console.

Cut boxing in half, and moved Courtroom into sliced half so it is now connected to security

Added a surgery room to medical, for future updates.

Added protolathes to both Epi and Engineering.

Added grinder and Hotplate to Epi

Added observation deck for South solar's (Also expanded maintenance for observation deck)

Psychologists office relocated to Reporters room.

Reporters room moved too old psychologists office.

Musicians room moved down to give more room for psychologists office.

Moved the path to AI room into new southeast maintenance

Extended bar out slightly

Revamped all of Security

Added spice to maintenance

Added a Portal to Naria

Removed the old teachings of Jeff Hoppingheimer

### CODE REVIEW

**Comment by Nousagi37** [comment]
Wubba Lubba Dub-Dub

**Comment by Vordenburg** [comment]
Having all of those entries under the public changelog is excessive. Maps are not intended to have more than one line per public changelog and only for large or significant changes. This is also not the right way to use the :cl: changelog feature, since each line is missing a `fix:` `add:` et cetera bullet point.

It's good to document all the changes to a map, but keep the minute details out of the public changelog.

**Comment by IamVelcroboy** [comment]
One thing I would suggest is some tile variation, or carpet in the bar/stage area. Maybe Bar tile, or dark tile behind the bar, carpet in the walk-through, or even a different tile for the stage. Something to break up the plain wood a little bit.

Perma looks really nice.

File: Resources/Prototypes/Maps/ovni.yml [hunk 1]
```diff
@@ -45,3 +45,4 @@
         Cataloguer: [ 0, 1 ]
         Reporter: [ 0, 1 ]
         Psychologist: [ 0, 1 ]
+        Prisoner: [ 1, 1 ]
```
Comment (Elijahrane) [inline Resources/Prototypes/Maps/ovni.yml]: do not put a prisoner slot on low pop maps

**Comment by lleftTheDragon** [comment]
> Having all of those entries under the public changelog is excessive. Maps are not intended to have more than one line per public changelog and only for large or significant changes. This is also not the right way to use the 🆑 changelog feature, since each line is missing a `fix:` `add:` et cetera bullet point.
> 
> It's good to document all the changes to a map, but keep the minute details out of the public changelog.

You're right my bad, i had a hard time reading the fine print when posting the PR, Fixed.

**Pull request merged**